src/status_im/ui/screens/profile/group_chat/styles.cljs

```diff
@@ -9,7 +9,7 @@
   {:background-color (colors/alpha colors/blue 0.1)
    :border-radius    50})
 
-(def action-label
+(defn action-label [with-subtext?]
```

Comment:
Argument not used?

Reply:
Yes, it's just that list action label styles are fns now. Rename arg to `_`?

Reply:
Ah ok! Yes please! And maybe a comment so that no-one removes it by accident?

Reply:
Renamed and added a comment.